service-csw/pom.xml

```diff
@@ -47,6 +47,11 @@
             <artifactId>postgresql</artifactId>
             <scope>test</scope>
         </dependency>
+		<dependency>
+			<groupId>org.codehaus.jackson</groupId>
+			<artifactId>jackson-mapper-asl</artifactId>
+			<version>1.9.13</version>
```

Comment:
You should use this one:

<dependency>
    <groupId>com.fasterxml.jackson.core</groupId>
    <artifactId>jackson-core</artifactId>
     <version>${jackson.version}</version>
</dependency>